dotnet/samples/KernelSyntaxExamples/Example43_GetModelResult.cs

```diff
@@ -15,6 +15,7 @@
 // ReSharper disable once InconsistentNaming
 public static class Example43_GetModelResult
 {
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "By design")]
```

Comment:
We have ~30 occurences of SuppressMessage and ~100 of `#pragma warning disable` -- curious if folks have thoughts here. I'd like to lean to usage of `#pragma`.

Reply:
fixed